src/joint.dia.link.js

```diff
@@ -1446,7 +1436,7 @@ joint.dia.LinkView = joint.dia.CellView.extend({
         // if are simulating pointerdown on a link during a magnet click, skip link interactions
         if (evt.target.getAttribute('magnet') != null) return;
 
-        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this, 'pointerdown') : this.options.interactive;
+        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this) : this.options.interactive;
```

Comment:
Can we get rid of this check here and again introduce a new feature? This time e.g `toolsUse` and use it [here](https://github.com/clientIO/joint/blob/master/src/joint.dia.link.js#L1500-L1505) instead?